### PR TITLE
Opsworks: Update application source setting after successful deploy

### PR DESCRIPTION
**Pull request by dazz**
Adding documentation on how to use the new option for updating the revision in OpsWorks App Settings after successful deployment.

PR https://github.com/travis-ci/dpl/pull/783

### CODE REVIEW

**Comment by BanzaiMan** [comment]
Thank you!!

File: user/deployment/opsworks.md [hunk 1]
```diff
@@ -111,6 +111,23 @@ deploy:
...
 
+### Updating App Settings after successful Deployments
+
+By default the deploy from Travis CI triggers a deployment on OpsWorks but does not touch any other configuration. To also update the revision in App Settings use the **update-app-on-success** option. In addition you have to set the **wait-until-deployed** option:
+
+```yaml
+deploy:
+  provider: opsworks
+  access-key-id: ACCESS-KEY-ID
+  secret-access-key: SECRET-ACCESS-KEY
```
Comment (hmaad30) [inline user/deployment/opsworks.md]: 054 653 8719

File: user/deployment/opsworks.md [hunk 2]
```diff
@@ -111,6 +111,23 @@ deploy:
...
+  app-id: APP-ID
+  wait-until-deployed: true
+  update-app-on-success: true
+```
+{: data-file=".travis.yml"}
+
+Travis CI will wait until the deployment returns successful and only then update the revision in App Settings.
+
 ### Conditional releases
```
Comment (hmaad30) [inline user/deployment/opsworks.md]: ###

Comment (hmaad30) [reply user/deployment/opsworks.md]: ###

**Pull request merged**